WireUI/Sources/WireAccountImageUI/AccountImageView/NotificationBadgeView.swift

```diff
@@ -0,0 +1,169 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+
+// MARK: -
+
+final class NotificationBadgeView: UIView {
+
+    // MARK: - Constants
+
+    enum Sizes {
+        static let backgroundBorderWidth: CGFloat = 1
+    }
+
+    enum Defaults {
+        static let badgeColor: UIColor = ColorTheme.NotificationBadge.fill
+        static let backgroundViewColor: UIColor = .systemBackground
+    }
+
+    // MARK: - Properties
+
+    var availability: Availability? {
+        didSet { setNeedsLayout() }
+    }
+
+    var badgeColor: UIColor = Defaults.badgeColor {
+        didSet { setNeedsLayout() }
+    }
+
+    var backgroundViewColor: UIColor = Defaults.backgroundViewColor {
+        didSet { backgroundView.backgroundColor = backgroundViewColor }
+    }
+
+    var showNotifications: Bool = false {
+        didSet { setNeedsLayout() }
+    }
+
+    // MARK: - Private Properties
+
+    /// A view which serves as background and outer border.
+    private let backgroundView = UIView()
+
+    /// An image view for the icon
+    private let iconView = UIImageView(image: .info)
```

Comment:
not urgent now, but in my opinion we should get rid of the extensions to `UIImage` and `Image`
```
public extension UIImage {
    static let infoCircle = UIImage(resource: .infoCircle)
}
```
because they almost don't add any value:
```suggestion
    private let iconView = UIImageView(image: .init(resource: .info))
```

Reply:
Even worse for the SwiftUI variant:
```
let infoCircle = Image.infoCircle
```
vs
```
let infoCircle = Image(.infoCircle)
```